thermal_processing.py

```diff
@@ -126,15 +126,20 @@ def one_candidate(candidates):
 def replace_ext(filename, ext):
     return filename.parent / (filename.stem + ext)
 
+
 def update_metadata(recording, api):
     with open(str(recording["filename"]), "rb") as f:
         reader = CPTVReader(f)
         metadata = {}
         metadata["recordingDateTime"] = reader.timestamp.isoformat()
+
         # TODO Add device name when it can be processed on api server
         # metadata["device_name"] = reader.device_name
 
-        count = 0.0
+        if reader.preview_secs:
+            metadata["additionalMetadata"] = {"previewSecs": reader.preview_secs}
+
+        count = 0
```

Comment:
Change from float to int.   Does the calculation still work?

Reply:
Yep. Python 3 will switch to floats automatically if the result of a division requires it.